lib/zabbix/zabbix_test.go

```diff
@@ -173,3 +174,35 @@ func TestCreateClientUsingAuth(t *testing.T) {
 		t.Errorf("expected client to be %v but got %v", expected, a)
 	}
 }
+
+func TestAcknowledgeEvent(t *testing.T) {
+	expected := map[string][]int{"eventids": []int{49}}
```

Comment:
How about the other case with multiple `eventids`?
I think the test should also stress that the first `eventid` in the slice.

Reply:
For now the focus is on one eventid, because we do not run multiple jobs at once for a trigger.